src/sagemaker/lineage/query.py

```diff
@@ -201,194 +201,101 @@ def _artifact_to_lineage_object(self):
-                                "font-family": "verdana",
-                            },
-                        },
-                        {"selector": ".select", "style": {"border-opacity": "0.7"}},
-                    ]
-                    + [self._create_entity_selector(k, v) for k, v in self.graph_styles.items()],
-                    responsive=True,
-                ),
-                self.html.Div(
-                    style={
-                        "width": "0.5%",
-                        "display": "inline-block",
-                        "font-size": "1vw",
-                        "font-family": "verdana",
-                        "vertical-align": "top",
-                    },
-                ),
-                # legend section
-                self.html.Div(
-                    [self._create_legend_component(v) for k, v in self.graph_styles.items()],
-                    style={
-                        "display": "inline-block",
-                        "font-size": "1vw",
-                        "font-family": "verdana",
-                        "vertical-align": "top",
-                    },
-                ),
-            ]
-        )
-
-        @app.callback(
-            self.Output("cytoscape-graph", "elements"),
-            self.Input("cytoscape-graph", "tapNodeData"),
-            self.Input("cytoscape-graph", "elements"),
-        )
-        def selectNode(tapData, elements):
-            for n in elements:
-                if tapData is not None and n["data"]["id"] == tapData["id"]:
-                    # if is tapped node, add "select" class to node
-                    n["classes"] += " select"
-                elif "classes" in n:
-                    # remove "select" class in "classes" if node not selected
-                    n["classes"] = n["classes"].replace("select", "")
+        return pyvis, Network, Options
 
-            return elements
+    def _get_options(self):
+        """Get pyvis graph options."""
+        options = """
+            var options = {
+            "configure":{
+                "enabled": false
+            },
+            "layout": {
+                "hierarchical": {
+                    "enabled": true,
+                    "blockShifting": true,
+                    "direction": "LR",
+                    "sortMethod": "directed",
+                    "shakeTowards": "leaves"
+                }
+            },
+            "interaction": {
+                "multiselect": true,
+                "navigationButtons": true
+            },
+            "physics": {
+                "enabled": false,
+                "hierarchicalRepulsion": {
+                    "centralGravity": 0,
+                    "avoidOverlap": null
+                },
+                "minVelocity": 0.75,
+                "solver": "hierarchicalRepulsion"
+            }
+        }
+            """
+        return options
 
-        return app
+    def _node_color(self, n):
```

Comment:
Can we use a more descriptive variable name than n? Also it might be better to pass in the value at `n[2]` instead of `n`